.github/workflows/tests.yml

```
@@ -7,17 +7,48 @@ on:
+      - 4.x
   pull_request:
 
 jobs:
   tests:
     runs-on: ubuntu-22.04
 
     strategy:
-      fail-fast: true
+      fail-fast: false
       matrix:
         php: [ 8.4 ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)